tests/periph_systick/app.config.test

```diff
@@ -0,0 +1,5 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=y
+CONFIG_MODULE_PERIPH_SYSTICK=y
+CONFIG_MODULE_PERIPH_SYSTICK_PRESCALER=y
```

Comment:
These symbols need to be defined, [here](https://github.com/RIOT-OS/RIOT/blob/master/drivers/periph_common/Kconfig) or on its own file in that folder.

Reply:
Can I also put this to `cortexm_common/Kconfig`?
How do I reproduce the Kconfig build locally?

Reply:
> Can I also put this to `cortexm_common/Kconfig`?

Hmm yeah, being this peripheral cortexm-specific that makes sense. I think it should not be a problem to place the symbols there there.

> How do I reproduce the Kconfig build locally?

To reproduce what murdock does:
` /bin/bash -c "source .murdock; JOBS=4 compile tests/periph_systick samr21-xpro:gnu"`

To just compile using Kconfig for module selection:
`BOARD=samr21-xpro TEST_KCONFIG=1 make -C tests/periph_systick`